File: remote/src/main/java/com/kelsos/mbrc/presenters/LibraryActivityPresenterImpl.java
package com.kelsos.mbrc.presenters;

import com.google.inject.Inject;
import com.kelsos.mbrc.interactors.LibraryStatusInteractor;
import com.kelsos.mbrc.ui.views.LibraryActivityView;

public class LibraryActivityPresenterImpl implements LibraryActivityPresenter {
  private LibraryActivityView view;

  @Inject private LibraryStatusInteractor libraryStatusInteractor;

  @Override public void bind(LibraryActivityView view) {

    this.view = view;
  }

  @Override public void onResume() {

  }

  @Override public void onPause() {

  }

  @Override public void checkLibrary() {

  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/presenters/LibraryActivityPresenterImpl.kt
package com.kelsos.mbrc.presenters

import com.kelsos.mbrc.ui.views.LibraryActivityView
import javax.inject.Inject

class LibraryActivityPresenterImpl
@Inject constructor() : LibraryActivityPresenter {
  private var view: LibraryActivityView? = null

  override fun bind(view: LibraryActivityView) {

    this.view = view
  }

  override fun onResume() {

  }

  override fun onPause() {

  }

  override fun checkLibrary() {

  }
}
